examples/LaserWakefield/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -140,6 +140,7 @@ struct Hydrogen
  * For development purposes: ---------------------------------------------
```

Comment:
this comment needs to be added to the general `src/picongpu/include/simulation_defines/param/speciesDefinition.param` file - but we can postpone this until ionization leaves the *in development* state.